index.html

```diff
@@ -1,12 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet">
 </head>
+
 <body>
-    
+    <header>
+        <h1>Weather Report</h1> <span>For the city of </span> <span id="headerCityName">Seattle</span>
+    </header>
+    <div class="wrapper">
+        <div class="box temp">
```

Comment:
Nice multi-class style. This lets you have the majority of the styling in a common class, while having the specific style in the dedicated class.